### DISCUSSION TITLE
How do I get the available updates to show like in the splash screen?

### DISCUSSION THREAD

**Discussion by z0rg0n** [BassT23/Proxmox#156] [Q&A] [answered]
This seems like it should be a super basic task. Maybe that's why I'm not finding it in the help or documentation.

I want to show this:
![image](https://github.com/user-attachments/assets/cb2db503-f057-4694-abb9-bfcd6abc1900)

Without having to start a new shell where it shows automatically.

I've tried each of the commands in --h and they don't seem to do it.

**Comment by BassT23** [comment]
Hi @z0rg0n,

this is the "Welcome-Screen".

Best would be: 

- first of all save your config from config file.
- uninstall updater with ```update uninstall```
- reinstall with ```bash <(curl -s https://raw.githubusercontent.com/BassT23/Proxmox/master/install.sh)```

during install you will be asked if you wan to install "Welcome-Screen" also ;) (with or without [neofetch](https://github.com/dylanaraps/neofetch))

**Reply by z0rg0n** [reply]
I'm sorry, I'm not being clear I think. I have the welcome screen installed and I like it a lot!

I like it so much I want to call it up in an empty terminal but I can't find the command for that

**Reply by BassT23** [reply, marked as answer]
for this, you could easely relogin to the node.
Then it would show up.

**Reply by BassT23** [reply]
its a file, created with update-checker. Stored in /etc/ultimate-updater/check-output

**Reply by z0rg0n** [reply]
That's perfect, thank you!
